# File: waywiser_perception/msg/DetectionArray.msg

std_msgs/Header header
waywiser_perception/Detection[] detections
